frontend/src/machines/billings/billingMachine.ts

```diff
@@ -247,6 +260,7 @@ export const billingMachine = createMachine<
   schema: {
     context: {} as Context,
     events: {} as MachineEvents,
+    services: {} as MachineService,
   },
```

Comment:
Ini kayaknya kalau pake [Typestates](https://xstate.js.org/docs/guides/typescript.html#typestates) gak perlu passing schema lagi ?

Reply:
Sorry, gak perlu nya karena kenapa ya?
Karena udah narrow down the shape of the overall state context based on the state value??

Reply:
iya betul, kalau kamu cek di contoh yang [Typestates](https://xstate.js.org/docs/guides/typescript.html#typestates), dia juga gak pake schema